src/components/30-organisms/modal/child.scss

```diff
@@ -0,0 +1,19 @@
+axa-modal[forced] > * {
+  &:first-child {
+    // !important reason: An axa-heading child element sets margins on the host
+    // element. This can only be overridden by !important.
+    // This is harmless because: There is no scenario where it would make sense
+    // for the first element to have any margin-top, except if they work with
+    // negative margins, which is an anti-pattern.
+    margin-top: 0 !important;
```

Comment:
Big change for very small outcome... its ok, i can see a difference.

Reply:
This is a axa-heading thing.. so maybe something like `&:first-child & axa-heading`? Ah, is that possible with versioning?!

Reply:
No this is also a thing for normal headings. It anyway makes sense to only use margin-top 0 in all cases, to prevent weird and unequal modals.

In my opinion, this change is not so big, because we use childelements, so it should be naturally, that we have styling for them in place.

Reply:
Or the modal do not use any spacings for themselfes. And the customer have to style the content. But jea, its ok.